castle/cms/interfaces/cache.py~

```diff
@@ -0,0 +1,17 @@
+from zope import schema
```

Comment:
I see your addition of *~ to .gitignore was that intended to exclude this file?

Reply:
Yes it was suppose to ignore all backup files that emacs automatically create.

Reply:
And removed.